doc/node-props.md

```diff
@@ -0,0 +1,11 @@
+# Properties of nodes
+These are properties babel stores in AST node objects for internal use, as opposed to properties that are part of the AST spec (ESTree at the time of this writing).
+
+## `_blockHoist`
+`node._blockHoist != null` triggers the [block-hoist transformer](/src/babel/transformation/transformers/internal/block-hoist.js). Value should be `true` or an integer in the range `0..3`. `true` is equivalent to `2`. The value indicates whether the node should be hoisted and to what degree. See the source code for more detailed information.
```

Comment:
@Mark-Simulacrum Did you click it from the rich diff? I don't know if that's a GitHub bug or expected, but it's resolving the link relative to the babel repo but with the branch name that only exists in my fork. You can see that it works in my [fork]( https://github.com/jmm/babel/blob/internals-docs2/doc/node-props.md), and it's a relative link, so yes, I think it'll resolve if it's merged here. If you don't think that explains the problem though, please let me know.